pkg/controller/ingress_worker.go

```diff
@@ -47,20 +47,21 @@ func (c *IngressController) runWatcher() {
 					c.enqueue(ing)
 				}
 			}
-		default:
+		default: // [review]: why have default here?
```

Comment:
I'm polling here - I don't know how to improve it. Without the default, I will never be able to check c.queue.ShuttingDown() and stop the watcher.